qa/L0_decoupled/decoupled_test.py

```diff
@@ -137,8 +137,8 @@ def _stream_infer_with_params(self, request_count, request_delay, _,
                     if response.parameters.get("triton_final_response"):
                       completed_requests += 1
 
-                    # Detect flags-only response, no need to process it
-                    if not response.parameters.get("triton_empty_response"):
+                    # Detect empty response, only process response if non-empty
+                    if response.outputs:
```

Comment:
`is None`?

Reply:
It's correct as-is, I clarified the comment. It's equivalent to checking that `response.outputs is not None` - i.e. non-empty response.

Reply:
I meant `is not None`, it's more for style, similar to `if (ptr)` v.s. `if (ptr != nullptr)`.

Reply:
`if response.outputs is not None` will return True for an empty list `[]`. 

`if response.outputs` will return True for non-empty list, it will check for "trueness" across types, like a list having length > 0.

Reply:
When will each of the cases occurs? If there are two different representations, they should have different meaning (even only slightly) to the user then?

Reply:
Discussed offline. Pythonic way is to just check "truthyness"/"falsiness". If we will assign different meanings to `[ ]` vs. `None` in the future, we can change it. If we want to enforce more strict type checking, we can do so in the client library in the future (ex: `assert isinstance(response.outputs, list)` etc.) - but the pythonic UX is to just check `if response.outputs` in this context.